xbmc/windowing/wayland/Seat.cpp

```diff
@@ -167,6 +170,13 @@ void CSeat::HandleKeyboardCapability()
   };
   m_keyboard.on_key() = [this](std::uint32_t serial, std::uint32_t time, std::uint32_t key, wayland::keyboard_key_state state)
   {
+      CLog::Log(LOGINFO, "key: {}, state: {}", key, (int)state);
+#ifdef TARGET_WEBOS
+      // Ignore webOS cursor show/hide mock keycodes
+      if (key == 1199 || key == 1198) {
```

Comment:
I removed this specific code path as it was only needed when the back action was the fallback for all buttons. Since we handle that correctly now it's not an issue anymore

Reply:
While this wasn't triggering the back action constantly anymore I noticed some UI flicking. I've changed the code above so that the XKB Keymap would actually map the cursor show/hide actions to unknown.